### ISSUE TITLE
onClose event is not triggered when clicking on the modal overlay

### ISSUE THREAD

**Issue by migliori** [ghosh/Micromodal#499]
Hi,

Thanks for this wonderful tool.

Here's my issue:

```
MicroModal.init({
    // ...
    disableScroll: true,
    onClose: modal => {
        document.body.style.overflow = '';
        console.log(document.body.style);
    }
});
```

When I click on the modal overlay it closes the modal, but the _onClose_ event is not triggered, so the document.body still has overflow: hidden

Thanks for your help

**Comment by kalpeshsingh** [comment]
Hello @migliori 

See if this [example](https://codesandbox.io/s/silly-burnell-wzsm6r?file=/src/App.js) can help you.

**Comment by migliori** [comment]
Hello @[kalpeshsingh](https://github.com/kalpeshsingh),

Thanks for your answer, I tried your example but the modal doesn't open and there's nothing in the console.

**Comment by kalpeshsingh** [comment]
Hello @migliori 
I'm not sure why didn't it work. 

You may want to try this one - https://codesandbox.io/s/cranky-microservice-7ctcgz

**Comment by migliori** [comment]
@[kalpeshsingh](https://github.com/kalpeshsingh),

Thanks but it still doesn't work

**Comment by kalpeshsingh** [comment]
Please share your example.
It will help us to debug effectively.

**Comment by migliori** [comment]
My sincere apologies for wasting your time.
I made a pen here: https://codepen.io/migli/pen/KKQxjxx and it worked exactly as expected.
When closing the modal by clicking the overlay the onClose event is triggered.

So I went back to my page at http://phpformbuilder/templates/bootstrap-5-forms/ajax-loaded-multiple-forms.html
and it works perfectly too now.

It can't be a browser cache issue (I always work without cache), so I don't know what happened, but your script is perfect, and I bothered you for nothing.

Thanks for your time and your excellent Micromodal
